### PR TITLE
treewide: remove redundant quotes

### PR DESCRIPTION
**Pull request by volth**
remove redundant quotes

I suspect it might require to adjust option's type somewhere (`types.str` -> `types.package`) but I did not hit the issue.

### CODE REVIEW

**Comment by rnhmjoj** [comment]
@volt, since you are doing some major cleaning... I noticed another inconsistency you may be interested in: many NixOS option descriptions contain urls not declared with `<link xlink:href="urlhere"/>`. It could probably be done with a simple regex search and replace.

File: pkgs/applications/audio/AMB-plugins/default.nix [hunk 1]
```diff
@@ -24,7 +24,7 @@ stdenv.mkDerivation rec {
     longDescription = ''
       Mono and stereo to B-format panning, horizontal rotator, square, hexagon and cube decoders.
     '';
-    version = "${version}";
+    version = version;
```
Comment (alyssais) [inline pkgs/applications/audio/AMB-plugins/default.nix]: While we’re here, could change these to `inherit`?

**Comment by alexarice** [comment]
just to check, I assume this is ignoring cases of the form `{ ... }."${x}"`

**Comment by volth** [comment]
> just to check, I assume this is ignoring cases of the form `{ ... }."${x}"`

because there is no such cases after https://github.com/NixOS/nixpkgs/pull/67536 :)

File: pkgs/applications/audio/AMB-plugins/default.nix (hunk 1 above)
Comment (volth) [reply pkgs/applications/audio/AMB-plugins/default.nix]: I would make it as a separate rule, there could be more `version = version;` case

**Comment by alexarice** [comment]
> 
> 
> > just to check, I assume this is ignoring cases of the form `{ ... }."${x}"`
> 
> because there is no such cases after #67536 :)

I didn't even realise that worked

**Comment by volth** [comment]
rebased

File: nixos/modules/services/mail/roundcube.nix [hunk 2]
```diff
@@ -132,13 +132,13 @@ in
         "listen.owner" = "nginx";
         "listen.group" = "nginx";
         "listen.mode" = "0660";
-        "pm" = "dynamic";
+        pm = "dynamic";
```
Comment (FRidh) [inline nixos/modules/services/mail/roundcube.nix]: while correct it is inconsistent here

**Comment by volth** [comment]
rebased merge conflict

File: nixos/modules/services/audio/roon-server.nix [hunk 3]
```diff
@@ -61,8 +61,8 @@ in {
     };
 
     
-    users.groups."${cfg.group}" = {};
-    users.users."${cfg.user}" =
+    users.groups.${cfg.group} = {};
```
Comment (FRidh) [inline nixos/modules/services/audio/roon-server.nix]: this is not correct

Comment (volth) [reply nixos/modules/services/audio/roon-server.nix]: > this is not correct

Why?

**Comment by volth** [comment]
rebased merge conflict

File: nixos/modules/services/audio/roon-server.nix (hunk 3 above)
Comment (FRidh) [reply nixos/modules/services/audio/roon-server.nix]: Ah, nevermind. I thought that was actually invalid syntax but it works.

**Comment by volth** [comment]
Rebased MC

**Comment by volth** [comment]
rebased mc

**Pull request merged**